.fixtures.yml

```diff
@@ -2,7 +2,9 @@ fixtures:
   repositories:
     stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
     inifile: "https://github.com/puppetlabs/puppetlabs-inifile.git"
-    archive: "https://github.com/puppet-community/puppet-archive.git"
+    archive:
+      repo: "git://github.com/voxpupuli/puppet-archive.git"
```

Comment:
I add my another email address.
If the latest puppet-archive module is used, rspec tests break !

I see another module with pin archive : 
https://github.com/puppetlabs/puppetlabs-java/blob/master/.fixtures.yml

Otherwise all the other PR for this module are break ...